lib/erlef/affiliate.ex

```diff
@@ -0,0 +1,5 @@
+defmodule Erlef.Affiliate do
```

Comment:
Instead of delegating and having a separate module we should probably just reach out to the Community context to get what we need. While, their will be separate pages (community and affiliates) it's not clear what the context for affiliates should belong to yet. Right now it makes the most sense I believe for it to belong to the Community context.